private/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package private contains helpers private to github.com/containers/image, with no intent or commitment to make them useful for any other purpose.
+// Do not use outside!
+package private
```

Comment:
Sure, will do. `pkg/private/explicitfilepath` and `pkg/private/dockerpolicyconfiguration` ?  Or do you want it without the `private` component?